src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/HandlerResolver.cpp

```diff
@@ -136,8 +136,8 @@ HandlerResolver::GetApplicationFactory(const IHttpApplication& pApplication, std
             errorContext.detailedErrorContent = to_multi_byte_string(format(ASPNETCORE_EVENT_MIXED_HOSTING_MODEL_ERROR_MSG, pApplication.GetApplicationId(), options.QueryHostingModel()), CP_UTF8);
             errorContext.statusCode = 500i16;
             errorContext.subStatusCode = 34i16;
-            errorContext.generalErrorType = "ASP.NET Core Module Mixed Hosting Models Not Supported";
-            errorContext.errorReason = "Select a different application pool to create another application.";
+            errorContext.generalErrorType = "ASP.NET Core does not support mixing hosting models";
+            errorContext.errorReason = "Select a different app pool to host this app.";
```

Comment:
Change all instances of app/apps to application/applications

Reply:
@anurse did this because our docs folks use app instead of application. I don't have a preference though.